.github/workflows/release.yaml

```diff
@@ -15,6 +15,91 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+
+  build-linux-arm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.21.x'
+          cache: false
+
+      - name: Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install build-essential ffmpeg protobuf-compiler ccache
+          sudo apt-get install -qy binutils-aarch64-linux-gnu gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+      - name: Install CUDA Dependencies
+        run: |
+          curl -O https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/cross-linux-aarch64/cuda-keyring_1.1-1_all.deb
+          sudo dpkg -i cuda-keyring_1.1-1_all.deb
+          sudo apt-get update
+          sudo apt-get install -y cuda-cross-aarch64
+        env:
+          CUDA_VERSION: 12-3
```

Comment:
nvidia isn't publishing cuda libraries for 12.3 for some reason, this need to be `12-4` or `12-5`

Reply:
we can start to think also to bundle that as part of the binary and then load libraries from there in runtime, that way we skip this annoying bit for users who are going to use static binaries

Reply:
some of that logic implemented in https://github.com/mudler/LocalAI/pull/2490/commits/44120364c3aa17d5d5627b12e41e2d0c7468a3ff - it should allow us to carry over libs. I've just tried the arm64 builds and "almost" works in my env. llama.cpp doesn't start as there is a libstdc++ incompatibility with the host